bucket/graalvm.json

```diff
@@ -1,22 +1,25 @@
 {
     "description": "High-performance, embeddable, polyglot Virtual Machine for JVM-langs (Java, Scala, Kotlin), JavaScript/NodeJS, Python, Ruby, R, and LLVM-langs (C, C++, Rust)",
-    "version": "20.3.0",
+    "version": "21.0.0.2",
     "homepage": "https://graalvm.org",
     "license": "GPL-2.0",
-    "url": "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-20.3.0/graalvm-ce-java11-windows-amd64-20.3.0.zip",
-    "hash": "f27b950c51e8297453e89fbd674b2efb7d6f5744419619537a6e84d662659c7b",
-    "extract_dir": "graalvm-ce-java11-20.3.0",
+    "url": "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-21.0.0.2/graalvm-ce-java11-windows-amd64-21.0.0.2.zip",
+    "hash": "fd74127557f05f5a62c71d681aba8cacc7dbd49359458c0981d80823ae2e2dfb",
+    "extract_dir": "graalvm-ce-java11-21.0.0.2",
     "env_add_path": "bin",
     "env_set": {
         "JAVA_HOME": "$dir",
         "GRAALVM_HOME": "$dir"
     },
     "checkver": {
-        "url": "https://www.graalvm.org/downloads/current-release/",
-        "regex": "graalvm-ce-builds/releases/tag/vm-([\\d\\w.-]+)"
+        "url": "https://github.com/graalvm/graalvm-ce-builds/releases",
+        "regex": "graalvm-ce-builds/releases/tag/vm-(21[\\d\\w.-]+)"
     },
     "autoupdate": {
         "url": "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-$version/graalvm-ce-java11-windows-amd64-$version.zip",
```

Comment:
Maybe update this so Java version is also captured, next Java LTS is around the corner, I presume they will make binaries for that too. If the user want to force Java 11, there is the -jdk11 manifest for that?